kamyroll_gui/utils/api.py

```diff
@@ -132,17 +130,16 @@ def _handle_error(code, message, use_login, channel_id):
 
         case "bad_player_connection":
             wait(2000)
-            return
 
         case "bad_initialize":
             wait(2000)
-            return
-
-        case _:
-            raise ApiError(f"Error: {code}. The provided id of the url is not valid")
 
+        # this is the same as `if code in ["unknown_id"]: ...`
+        case x if x in ["unknown_id"]:
```

Comment:
```suggestion
        case "unknown_id":
```
There is one element in the list anyways, so it makes sense to match the exact case. This also removes the need for a comment